### PR TITLE
Bootstrap composite ci-lint action

### PR DESCRIPTION
**Pull request by K-Phoen**

### CODE REVIEW

File: release-production/action.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      shell: bash
+      env:
+        DOCKER_TOOLS: true
+
+    - run: TAG_NAME=$(basename ${GITHUB_REF}) make trigger_spinnaker
+      shell: bash
+      env:
+        SPINNAKER_SECRET: ${{ inputs.spinnaker-secret }}
+
+    - run: SWAGGER_HUB_TOKEN="${{ secrets.SWAGGER_HUB_TOKEN }}" VERSION=$(basename ${GITHUB_REF}) [ -z "$SWAGGER_HUB_TOKEN" ] || make publish_docs
```
Comment (ronanbarrett) [inline release-production/action.yml:34]: Is this correct `secrets.SWAGGER_HUB_TOKEN` ? Shouldn't this be sent as input too? i.e. `input.swagger-hub-token`

Comment (K-Phoen) [reply release-production/action.yml]: Yep, my bad! Nice catch :)

**Review by ronanbarrett — DISMISSED** [review]

File: release-production/action.yml (hunk 1 above)
Comment (ronanbarrett) [reply release-production/action.yml]: Maybe add to README too?

**Review by ronanbarrett — APPROVED** [review]
👍

**Pull request merged**